### PR TITLE
Added comments to code

### PR DESCRIPTION
**Pull request by InfernoCoder**
According to the GCI task, I have added comments to the code.

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 1]
```diff
@@ -50,6 +55,7 @@ protected void onCreate(Bundle savedInstanceState) {
         ImageView faceView = (ImageView) findViewById(R.id.faceView);
         final ImageView hairView = (ImageView) findViewById(R.id.hairView);
         final ImageView clothView = (ImageView) findViewById(R.id.clothView);
+        // Fetch character data (eye, face etc.) from Database
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Remove the very generic comments

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 2]
```diff
@@ -197,6 +203,7 @@ public void onClick(View v) {
             tv.setText(R.string.hair);
             ImageButton left = (ImageButton) findViewById(R.id.leftSelectFeature);
             ImageButton right = (ImageButton) findViewById(R.id.rightSelectFeature);
+            // Move left and select an item
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Remove the obvious comments

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarActivity.java [hunk 3]
```diff
@@ -21,6 +25,8 @@ public void onCreate(Bundle savedInstanceState) {
         setmDbHandler(new DatabaseHandler(this));
         getmDbHandler().open();
         fromActivity = getIntent().getExtras().getInt(getResources().getString(R.string.from_activity));
+
+        //Initialize the elements in the activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarActivity.java]: Remove generic comments

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 4]
```diff
@@ -36,6 +41,7 @@ public void onCreate(Bundle savedInstanceState) {
         getmDbHandler().open();
         avatarRoomInstance = this;
         setContentView(R.layout.avatar_room);
+        // Initialize elements in Activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 5]
```diff
@@ -54,6 +60,7 @@ public void onCreate(Bundle savedInstanceState) {
         ImageButton hairRight = (ImageButton) findViewById(R.id.hairRight);
         Button continueButton = (Button) findViewById(R.id.continueButtonAvatar);
 
+        //region Contains OnClickListeners to cycle between and select the required features like face, eyes, etc.
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Remove

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 6]
```diff
@@ -233,10 +240,13 @@ public void onClick(View v) {
                 }
             }
         });
+        //endregion
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 7]
```diff
@@ -23,11 +27,14 @@ protected void onCreate(Bundle savedInstanceState) {
         setmDbHandler(new DatabaseHandler(this));
         getmDbHandler().open();
         dressingRoomInstance = this;
+        // Initialize elements in Activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: Same.

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 8]
```diff
@@ -95,7 +102,7 @@ protected void onCreate(Bundle savedInstanceState) {
         ImageView hairImageView = (ImageView) findViewById(R.id.hairImageView);
         ImageView accessoriesImageView = (ImageView) findViewById(R.id.accessoriesImageView);
 
-
+        // OnClickListeners to change things like dress, hair or accessories.
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: Remove

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 9]
```diff
@@ -48,6 +52,7 @@ public void onCreate(Bundle savedInstanceState) {
         listAdapter = new ArrayAdapter<>(this, R.layout.simplerow,
                 new ArrayList<String>());
         answers = new ArrayList<>();
+        // Initialize elements in Activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 10]
```diff
@@ -66,7 +72,6 @@ public void onCreate(Bundle savedInstanceState) {
             // TODO Auto-generated catch block
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Remove this comment from every part of the code

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 11]
```diff
@@ -10,10 +10,16 @@
...
 public class MapActivity extends Activity {
 
+    /**
+     * The base map from which a Scenario is played
+     */
+
     private DatabaseHandler mDbHandler;
     private OnClickListener onClickListener = new OnClickListener() {
         @Override
         public void onClick(View v) {
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: Remove

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 12]
```diff
@@ -23,16 +29,14 @@ public void onClick(View v) {
...
-     * Called when the activity is first created.
-     */
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setmDbHandler(new DatabaseHandler(this));
         getmDbHandler().open();
         setContentView(R.layout.gamemap);
 
+        // Initialize buttons from XML file and set their onClickListeners
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/ScenarioOverActivity.java [hunk 13]
```diff
@@ -27,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_scenario_over);
         scenarioOverActivityInstance = this;
         scenarioActivityDone = 1;
+        // Initialize elements in Activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/ScenarioOverActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 14]
```diff
@@ -39,6 +43,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setmDbHandler(new DatabaseHandler(this));
         getmDbHandler().open();
         selectFeatureInstance = this;
+        //Initialize the elements in the activity (XML file)
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Same

**Comment by InfernoCoder** [comment]
@anubhakushwaha  Should I delete this comment too - // Update character's characteristics from database?

**Comment by anubhakushwaha** [comment]
@InfernoCoder No

**Comment by InfernoCoder** [comment]
Ok

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java (hunk 11 above)
Comment (InfernoCoder) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: In this, remove which one?

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 15]
```diff
@@ -10,10 +10,16 @@
...
+    /**
+     * The base map from which a Scenario is played
+     */
+
     private DatabaseHandler mDbHandler;
     private OnClickListener onClickListener = new OnClickListener() {
         @Override
         public void onClick(View v) {
+            // This onClickListener is used to open a Scene
+            // and it is used as a global variable to remove copies of same code.
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: @InfernoCoder These

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java (hunk 11 above)
Comment (InfernoCoder) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: @anubhakushwaha Should I remove this one - The base map from which a Scenario is played?

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java (hunk 15 above)
Comment (InfernoCoder) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: Ok

**Comment by InfernoCoder** [comment]
@anubhakushwaha Modified the comments as required.

**Comment by anubhakushwaha** [comment]
@chhavip This can be closed as it is now a redundant PR .Fixed via #210

**Comment by chhavip** [comment]
Closing.

**Pull request closed without merging**